specs/OracleQueries.yml

```diff
@@ -437,32 +499,12 @@ components:
         type: integer
         format: int64
       example: "10"
-    forename:
-      name: 'forename'
-      description: The officers' first name.
-      in: 'query'
-      required: false
-      schema:
-        type: string
-    surname:
-      name: 'surname'
-      description: The officers' surname.
-      in: 'query'
-      required: false
-      schema:
-        type: string
-    dateOfBirth:
-      name: 'dateOfBirth'
-      description: The officers' date of birth.
-      in: 'query'
-      required: false
+    ephemeralOfficerKey:
+      name: 'ephemeral_officer_key'
+      description: The ephemeral key for a bankrupt officer. This is a disposable key which only lasts until the next daily upload.
+      in: 'path'
+      required: true
       schema:
         type: string
-      example: "2020-01-01"
-    postcode:
-      name: 'postcode'
-      description: The postcode for the address.
-      in: 'query'
-      required: false
-      schema:
-        type: string
\ No newline at end of file
+        format: uuid
+      example: "B67053D91CC750DEE05400144FFBDD12"
```

Comment:
If uuid ...is format not with hashes ?
eg
      format: uuid
      example: "b72596ce-da4e-41f9-bda5-2e52eeff8730"

Reply:
Good catch - specifically it's a guid so doesn't have quite the same conventions as a uuid - generated automatically via Oracle DB

Reply:
Changed to be a string from guid per conversation with @gn404